src/screens/SearchScreen.js

```diff
@@ -86,23 +86,21 @@ class SearchScreen extends Component {
     };
 
     renderSearchScreen = () => {
-        const {viewStyle, lastKnownRegion, mapLoaded} = this.props;
-
-        const something = lastKnownRegion;
+        const {viewStyle, lastKnownRegion, mapLoaded, sites} = this.props;
 
         if (viewStyle === map.SearchOptions.MAP) {
             return (
                 <SearchMap
                     lastKnownRegion={lastKnownRegion}
                     mapLoaded={mapLoaded}
                     updateRegion={this.props.updateRegion}
-                    sites={[]}
+                    sites={sites}
                 />
             );
         } else {
             return (
                 <SearchList
-                    sites={[]}
+                    sites={sites}
```

Comment:
Now actually passing in sites. When we get real sites, we'll just need to change the reducer; the rest of the app is ready to accept the data.